docs/rst/fastdds_qos_profiles_manager_cli/cli_elements/cli_common_elements/cli_common_duration_type_element.rst

```diff
@@ -30,6 +32,10 @@ The duration type configurable ``<subelements>`` are explained in the table belo
       - ``duration.<duration_type>``
       - Complex parameter
       - ``infinite``
+    * - All ``subelements``
```

Comment:
I am thinking to refactor a little this section. Properly speaking, only `sec` and `nanosec` are `subelements` and should be displayed in the table. The rest are supported usages but not properly speaking subelements, so they should not be displayed in the table.

Consequently, I suggest removing both this and the infinite case from the table and add a paragraph below. Something like this:

```
Duration type can also be set in the following supported usages:

* Setting an infinite duration: ``duration.<duration_type> infinite``
* Setting the duration directly: ``duration.<duration_type> <sec_value> <nanosec_value>``
```

I would also update the note below stating that:

```
The duration can be directly set also by passing only the ``<sec_value>`` argument: ``duration.<duration_type> <sec_value>``
```

What do you think?